addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -52,15 +52,30 @@ WebsiteNewMenu.include({
 odoo.define('website_sale.editor', function (require) {
 'use strict';
 
-require('web.dom_ready');
+var core = require('web.core');
+var Dialog = require('web.Dialog');
 var options = require('web_editor.snippets.options');
+var sAnimations = require('website.content.snippets.animation');
+
+var _t = core._t;
+var qweb = core.qweb;
+
+sAnimations.registry.websiteSaleCurrency = sAnimations.Class.extend({
```

Comment:
You can now use public widgets instead of animations (they are the same but more efficient, see publicWidget). Also, as this one is meant to be used in editable mode, you must add the `disabledInEditableMode` property to false though.